x/pocketcore/types/errors.go

```diff
@@ -411,8 +413,9 @@ func NewEmptyPayloadDataError(codespace sdk.CodespaceType) sdk.Error {
 	return sdk.NewError(codespace, CodeEmptyPayloadDataError, EmptyPayloadDataError.Error())
 }
 
-func NewInvalidHashError(codespace sdk.CodespaceType, err error) sdk.Error {
-	return sdk.NewError(codespace, CodeInvalidHashError, InvalidHashError.Error()+err.Error())
+func NewInvalidHashError(codespace sdk.CodespaceType, err error, h string) sdk.Error {
+	InvalidHash := fmt.Sprintf("%s %s is invalid due to: %s", InvalidHashError.Error(), h, err.Error())
```

Comment:
I'd like to move the error message to the variabel

then just sprintf `fmt.Sprintf("%s %s %s, InvalidHashError.Error(), h, err.Error())`

Reply:
`fmt.Sprintf("%s %s%s", ....)` since the msg error is `"the hash is invalid: "`